src/components/FilterableList/FilterableList.jsx

```diff
@@ -6,7 +6,12 @@ function FilterableList({ items, onChange, inputValue }) {
   return (
     <ul className={styles.list}>
       <li className={styles.listItem}>
-        <input type="text" className={styles.searchBox} onChange={onChange} />
+        <input
+          type="text"
+          className={styles.searchBox}
+          onChange={onChange}
+          placeholder={'Enter city name'}
```

Comment:
no need to use curly braces in this case

Reply:
for me, all string constant need to move to some const.